ci/pipelines/pr.yml

```diff
@@ -566,6 +566,7 @@ resources:
     icon: github-circle
     check_every: 1h
     source: &pull-request-source
+      disable_forks: true
```

Comment:
welp

Reply:
👋 byeeeee

Reply:
See #225 Tests run in a container env with elevated permissions for 🐳 in 🐳  reasons.

Reply:
I might be able to make that mostly go away... no promises though